plugins/rapid7_insight_agent/help.md

```diff
@@ -80,7 +80,8 @@ Example output:
 
 #### Get Agent Details
 
-This action is used to find and display detailed information about a device. If additional pages of agents are available, the action should be run again with the returned next cursor
+This action is used to find and display detailed information about a device. If additional pages of agents are 
+available, the action should be run again with the returned next cursor
```

Comment:
Can we reset the formatting on all these actions that have been spread over two lines back to one?